packages/fxa-settings/src/styles/tailwind.css

```diff
@@ -61,4 +61,14 @@ progress[value]::-moz-progress-bar {
   .break-word {
     word-break: break-word;
   }
+
+  /* This global setting ensures that all animated SVGs are paused if a user prefers reduced motion.
```

Comment:
See this PR's deployed storybook to test the reduced motion setting (set in OS accessibility settings)